test/unit/canvas_events.js

```diff
@@ -346,24 +346,6 @@
     assert.equal(prepareFor, true, 'the brush called the _setBrushStyles method');
   });
 
-  QUnit.test('mouse:up should return target and currentTarget', function(assert) {
-    var e1 = { clientX: 30, clientY: 30, which: 1, target: canvas.upperCanvasEl };
-    var e2 = { clientX: 100, clientY: 100, which: 1, target: canvas.upperCanvasEl };
-    var rect1 = new fabric.Rect({ left: 0, top: 0, width: 50, height: 50, lockMovementX: true, lockMovementY: true });
-    var rect2 = new fabric.Rect({ left: 75, top: 75, width: 50, height: 50 });
-    canvas.add(rect1);
-    canvas.add(rect2);
-    var opt;
-    canvas.on('mouse:up', function(_opt) {
-      opt = _opt;
-    });
-    canvas.__onMouseDown(e1);
-    canvas.__onMouseMove(e2);
-    canvas.__onMouseUp(e2);
-    assert.equal(opt.target, rect1, 'options match model - target');
-    assert.equal(opt.currentTarget, rect2, 'options match model - currentTarget');
-  });
```

Comment:
yes we had a specific test that say, should return. So it isn't casual, is something we added for a reason and if we want to remove it it should be at least its individual revertable PR.